app/routes/404.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react'
+import {ErrorPage} from '../components/errors'
 
 export function meta() {
   return {title: "Ain't nothing here"}
 }
 
-export default function FourOhFour() {
+export default function NotFoundPage() {
   return (
-    <div>
-      <h2>404</h2>
-    </div>
+    <ErrorPage
+      title="404 - Oh no, how did you get here?"
+      subtitle="This is not a page on kentcdodds.com. So sorry."
+    />
```

Comment:
This page seems to be a bit special to remix. If I would render `<FourOhFour>` here directly, it throws:

```shell
    You must render this element inside a <Remix> element

      30 |
      31 | function FourOhFour() {
    > 32 |   const matches = useMatches()
         |                   ^
      33 |   const last = matches[matches.length - 1]
      34 |   const pathname = last?.pathname
```